README.md

```diff
@@ -1,138 +1,221 @@
-# rapids-env-generator
+# rapids-dependency-file-generator
 
-`rapids-env-generator` is a Python package that generates multiple `conda` environment files from a single YAML file. When installed via `pip`, it makes the `rapids-env-generator` CLI command available which is responsible for parsing an `envs.yaml` configuration file and generating the appropriate `conda` environment files.
+`rapids-dependency-file-generator` is a Python CLI tool that generates `conda` environment files and `requirements.txt` files from a single YAML file, typically named `dependencies.yaml`. When installed, it makes the `rapids-dep-file-generator` CLI command available which is responsible for parsing a `dependencies.yaml` configuration file and generating the appropriate `conda` environment and `requirements.txt` dependency files.
+
+## Table of Contents
+
+- [Installation](#installation)
+- [Usage](#usage)
+- [`dependencies.yaml` Format](#dependenciesyaml-format)
+  - [`files` key](#files-key)
+  - [`channels` key](#channels-key)
+  - [`dependencies` key](#dependencies-key)
+- [How Dependency Lists Are Merged](#how-dependency-lists-are-merged)
+- [Additional CLI Notes](#additional-cli-notes)
+
+## Installation
+
+`rapids-dependency-file-generator` is available on PyPI. To install, run:
+
+```sh
+pip install rapids-dependency-file-generator
+```
 
 ## Usage
 
-By default, when `rapids-env-generator` is run, it will look for a `conda/environments/envs.yaml` file relative to the directory from which it's invoked. `rapids-env-generator` will then generate the corresponding environment files based on the contents of the `envs.yaml` file. By default, the generated environment files will be output to the same directory as the `envs.yaml` file (i.e. `conda/environments`).
+When `rapids-dep-file-generator` is invoked, it will read a `dependencies.yaml` file from the current directory and generate children dependency files. `dependencies.yaml` is intended to be committed to the root directory of repositories. It has specific keys (described below) that enable the bifurcation of dependencies for different CUDA versions, architectures, and dependency file types (i.e. `conda` environment files vs `requirements.txt`). The bifurcated dependency lists are merged according to the description in the [_How Dependency Lists Are Merged_](#how-dependency-lists-are-merged) section below.
```

Comment:
Let's use semantic line breaks here - one sentence per line. Much easier to review/edit (I just wanted to add a period after "vs." but had to suggest something for the entire block.)
```suggestion
When `rapids-dep-file-generator` is invoked, it will read a `dependencies.yaml` file from the current directory and generate children dependency files.
`dependencies.yaml` is intended to be committed to the root directory of repositories.
It has specific keys (described below) that enable the bifurcation of dependencies for different CUDA versions, architectures, and dependency file types (i.e. `conda` environment files vs. `requirements.txt`).
The bifurcated dependency lists are merged according to the description in the [_How Dependency Lists Are Merged_](#how-dependency-lists-are-merged) section below.
```